OpenSim/Simulation/Model/Model.cpp

```diff
@@ -1686,7 +1686,7 @@ void Model::writeMarkerFile(const string& aFileName)
  * @param aMarkerSet The new marker set to copy.
  * @return Number of markers that were successfully added to the model.
  */
-int Model::replaceMarkerSet(const SimTK::State& s, const MarkerSet& aMarkerSet)
+int Model::replaceMarkerSet(SimTK::State& s, const MarkerSet& aMarkerSet)
```

Comment:
The current workflow in the `GenericModelMaker` is to call `Model::updateMarkerSet(newMarkerSet)` which appears to be performing a union of the existing and new markers.  I don't think we should have both methods.